container/raw/factory.go

```diff
@@ -62,14 +66,24 @@ func (self *rawFactory) NewContainerHandler(name string, inHostNamespace bool) (
 // The raw factory can handle any container. If --docker_only is set to false, non-docker containers are ignored.
 func (self *rawFactory) CanHandleAndAccept(name string) (bool, bool, error) {
 	accept := name == "/" || !*dockerOnly
+
+	if *dockerOnly {
```

Comment:
instead of having this only in effect when `--docker_only` is specified, can we make it always take effect, but have the default be `[ "/" ]`, so all raw cgroups are collected?  We can make setting --docker_only set this to `[ ]` during creation of the raw factory.

Reply:
Sounds good to me. Will do.

Reply:
> We can make setting --docker_only set this to [ ] during creation of the raw factory.

We should not do this, imo. What we want is to respect with whitelist settings even `docker_only` is set.

Reply:
What i think is about this: 
* by default setting `rawPrefixWhiteList` to `[]string{"/"}` to allow all raw cgroups to be collected or be compatible with backwards
* always filter whitelist raw cgroups and respect with the whitelist settings.

Reply:
PTAL.